pkg/feature/domain/evaluation.go

```diff
@@ -117,7 +117,7 @@ func TopologicalSort(features []*featureproto.Feature) ([]*featureproto.Feature,
 		for _, p := range f.Prerequisites {
 			pf, ok := mapFeatures[p.FeatureId]
 			if !ok {
-				return errFeatureNotFound
+				return ErrFeatureNotFound
```

Comment:
Do you have a reason why we need to change from `errFeatureNotFound` to `ErrFeatureNotFound`?

Reply:
There is no need, thx!

Reply:
@ono-max 
I fixed it 👍